test/assets/unrunnableTestCases.csv

```diff
@@ -108,7 +108,7 @@ fn-avg-6,Error: FORG0006: items passed to fn:avg are not all numeric.
 fn-avg-10,Error: FORG0006: items passed to fn:avg are not all numeric.
 cbcl-avg-004,Error: FORG0006: items passed to fn:avg are not all numeric.
 cbcl-avg-006,Error: FORG0006: items passed to fn:avg are not all numeric.
-cbcl-avg-008,Error: XPST0003: Unable to parse XPath: "typeswitch (fn:avg((xs:float(1), xs:double(2), xs:float(3)))) case $x as xs:double return $x default return "FAIL"". Expected "!", "!=", "*", "+", ",", "-", "/", "//", "<", "<<", "<=", "=", "=>", ">", ">=", ">>", "[", "and", "cast", "castable", "div", "eq", "except", "ge", "gt", "idiv", "instance", "intersect", "is", "le", "lt", "mod", "ne", "or", "to", "treat", "union", "|", "||", or end of input but "(" found. typeswitch [Error is around here](fn:avg((xs:float(1), xs:double(2), xs:float(3)))) case $x as xs:double return $x default return "FAIL"
+cbcl-avg-008,Error: XPST0008, The variable x is not in scope.
```

Comment:
I understood from @DrRataplan that we have not yet implemented this, can we throw a not implemented error instead of claiming the script is wrong.

Reply:
Here, we have to check is the declared variable is being used on the return clause of every case inside the typeswitch, and I don't now if there's an easy way to do that. Any suggestions? @JosVerburg @DrRataplan